### PR TITLE
Add `MWC4K 2024` staff recruitment news post

### PR DESCRIPTION
**Pull request by LeoFLT**
Now with 100% less "prestigious tournament series" usage.

Includes a (somewhat) square aspect ratio flowchart to make things easier to understand

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: osu!mania 4k World Cup 2024 Mapper/Playtester Applications
```
Comment (Walavouchey) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:3]:
```suggestion
title: osu!mania 4k World Cup 2024 Mapper and Playtester Applications
```

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Playtesters will be responsible for playing and giving feedback for a plethora of beatmaps (as requested by the mappooling team) to gauge aspects such as difficulty, suitability for a certain slot, and/or skill representativity among others (specific workflows vary for each team). Playtesters are generally expected to have considerable skill on their respective game mode.
+
+Please note that if you are accepted into either team, you will not be eligible to participate as a player in the tournament.
+
+### Recruitment process
+
+![](/wiki/shared/news/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications/mwc4k2024-apps-flowchart.png)
+
+All prospective helpers will have to answer the first with form, which contains questions aimed at gauging whether or not they're a good fit for the current team.
```
Comment (Walavouchey) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:25]:
```suggestion
All prospective helpers will have to answer the first form, which contains questions aimed at gauging whether or not they're a good fit for the current team.
```

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Participants that clear the first stage will have to answer another form, this time with even more specific questions meant to gauge their experience, skill, and methodology on various aspects of mapping and/or playtesting. After this stage, one of four things may happen:
+
+  - The candidate is invited to staff for MWC 4K
+  - The candidate is invited to an individual interview before proceeding
+  - The candidate is invited to staff for RAFT
+  - The candidate is not invited to either team
```
Comment (Walavouchey) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:32]:
```suggestion
- The candidate is invited to staff for MWC 4K
- The candidate is invited to an individual interview before proceeding
- The candidate is invited to staff for RAFT
- The candidate is not invited to either team
```

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Please note that not all participants will receive invitations to the second and/or third stages, and may not receive invites even if they go through all steps. 
+
+If you are considering applying but do not feel like you have the necessary experience, we urge you to apply – the 4K MWC mappooling team alongside the osu!mania Tourney Central Council has developed a controlled-environment staff mentorship program under the Rapid Fire Tournament Series project (RAFT), which will give mappers and playtesters with less tournament staffing experience but high potential and interest an opportunity to demonstrate their ability before being fully accepted into the 4K MWC mappooling team.
```
Comment (Walavouchey) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:36]:
```suggestion
If you are considering applying but do not feel like you have the necessary experience, we urge you to apply — the 4K MWC mappooling team alongside the osu!mania Tourney Central Council has developed a controlled-environment staff mentorship program under the Rapid Fire Tournament Series project (RAFT), which will give mappers and playtesters with less tournament staffing experience but high potential and interest an opportunity to demonstrate their ability before being fully accepted into the 4K MWC mappooling team.
```

Comment (Walavouchey) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:36]: link RAFT

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/shared/news/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications/mwc4k2024-apps-flowchart.png
Comment (Hiviexd) [inline wiki/shared/news/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications/mwc4k2024-apps-flowchart.png:1]: would trim the whitespace on top and bottom of this pic, gaps feel a bit too much rn

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md (hunk 4 above)
Comment (LeoFLT) [reply news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:36]: RAFT is an invite-only thing, the form goes more in-depth but it's still a trial

Comment (Walavouchey) [reply news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md:36]: is there no information about it online?

Comment (Walavouchey) [reply news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md]: what about the tourney central council?

Comment (LeoFLT) [reply news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md]: > is there no information about it online?

It's something they created for these apps in specific, so not really

Comment (LeoFLT) [reply news/2024/2024-02-27-osumania-4k-world-cup-2024-mapperplaytester-applications.md]: link Tourney Central Council added on 220bacd

File: news/2024/2024-02-27-osumania-4k-world-cup-2024-mapper-and-playtester-applications.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- The candidate is invited to staff for RAFT
+- The candidate is not invited to either team
+
+Please note that not all participants will receive invitations to the second and/or third stages, and may not receive invites even if they go through all steps.
+
+If you are considering applying but do not feel like you have the necessary experience, we urge you to apply — the 4K MWC mappooling team alongside the [osu!mania Tourney Central Council](https://docs.google.com/spreadsheets/d/1Rmi_eiuZ855LNVSBH81Yv4BVHEpiT-Nll8UVyP6ToOY) has developed a controlled-environment staff mentorship program under the Rapid Fire Tournament Series project (RAFT), which will give mappers and playtesters with less tournament staffing experience but high potential and interest an opportunity to demonstrate their ability before being fully accepted into the 4K MWC mappooling team.
+
+---
+
+Did either position catch your interest? Make sure to fill the application form available **[here](https://forms.gle/3JK3BFb7ZQNWK8qP8)**. The World Cup team will forever be grateful for your future contributions to the project!
```
Comment (Hiviexd) [inline news/2024/2024-02-27-osumania-4k-world-cup-2024-mapper-and-playtester-applications.md:40]:
```suggestion
Did either position catch your interest? **[Make sure to fill the application form available here](https://forms.gle/3JK3BFb7ZQNWK8qP8)**. The World Cup team will forever be grateful for your future contributions to the project!
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**